### DISCUSSION TITLE
Choosing the right engine?

### DISCUSSION THREAD

**Discussion by linuxdep** [nliaudat/floor-heating-proportional-valve#3] [General]
What kind of engine did you use?
What RPM is the best, in your link there are multiple speeds to choose from?

Does the motor switch off with your controller at the end stop?

Do you have a small video about how the whole thing works?
The idea is really cool, it reduces the costs a bit for some actuators.

**Comment by nliaudat** [comment]
[GA12-N20-M3 15rpm 3V](https://fr.aliexpress.com/item/4000311295036.html?spm=a2g0s.12269583.0.0.18834b0ejbgKSj) ~3.5 $

15 RPM for the max torque
The endstop are controlled by [the controller board](https://github.com/nliaudat/esp32_8ch_motor_shield)
The project could be easily adapt to embed it's own endstops

I have no video but you can enable simulation mode in openscad
Regards
